source/parser.hera

```diff
@@ -547,6 +547,8 @@ NonNullAssertion
 # ie: the 'of' in 'for x of ...'
 AdditionalReservedWords
```

Comment:
I do wonder whether there's a better name than `AdditionalReservedWords`, like `ForbiddenImplicitCalls` or something. There's no reference to the other `ReservedWord` rule so it's not really additional.

Reply:
I'll update the name.